Makefile

```diff
@@ -10,18 +10,21 @@ export BUILD_RUNTIMES?=$(HOME)/.cache/hypothesis-build-runtimes
 export TOX_WORK_DIR=$(BUILD_RUNTIMES)/.tox
 export COVERAGE_FILE=$(BUILD_RUNTIMES)/.coverage
 
-PY27=$(BUILD_RUNTIMES)/snakepit/python2.7
-PY273=$(BUILD_RUNTIMES)/snakepit/python2.7.3
-PY33=$(BUILD_RUNTIMES)/snakepit/python3.3
-PY34=$(BUILD_RUNTIMES)/snakepit/python3.4
-PY35=$(BUILD_RUNTIMES)/snakepit/python3.5
-PY36=$(BUILD_RUNTIMES)/snakepit/python3.6
-PYPY=$(BUILD_RUNTIMES)/snakepit/pypy
+SNAKEPIT=$(BUILD_RUNTIMES)/snakepit
+
+PY27=$(SNAKEPIT)/python2.7
+PY273=$(SNAKEPIT)/python2.7.3
+PY33=$(SNAKEPIT)/python3.3
+PY34=$(SNAKEPIT)/python3.4
+PY35=$(SNAKEPIT)/python3.5
+PY36=$(SNAKEPIT)/python3.6
+PYPY=$(SNAKEPIT)/pypy
 
 BEST_PY3=$(PY36)
 
 TOOLS=$(BUILD_RUNTIMES)/tools
 
+VIRTUALENV_CMD=$(TOOLS)/virtualenv
```

Comment:
This `VIRTUALENV_CMD` bit looks like it might be something that could be split out into a separate patch?